client/components/mma/Page.tsx

```diff
@@ -170,15 +170,15 @@ const PageNavAndContentContainer = (props: PageNavAndContentContainerProps) => (
 				...gridItemPlacement(1, 4),
 
 				[from.tablet]: {
-					...gridItemPlacement(1, 12),
```

Comment:
Why have these changed? It will affect every page on MMA so may make some of the designs in journeys look a bit off

Reply:
I was asked by Dom Whooley to update the design. I can hold off on this and create a new PR.

Reply:
I've removed this and will create a new PR for this specific change

Reply:
Cool 👍